static/js/event/types.ts

```diff
@@ -17,3 +17,21 @@
 /**
  * Types specific to event pages
  */
+
+export interface PropertyValue {
```

Comment:
nit: this is a good candidate for a shared interface (since all callers of the triples api would need this). for future reference

Reply:
Moved to shared interface!